docs/_artifacts/document-attributes.adoc

```diff
@@ -3,41 +3,41 @@
 // Attributes MUST appear below the document title
 // Do not delete attributes with identical values, such as :product-long: and :product:. These are intentional because although they are identical here, they are not in other renderings of this source file.
 
-
+//General
 :org-name: Application Services
 :community:
 :imagesdir: ./images
-:product-version: 1
-:product-long: OpenShift Streams for Apache Kafka
-:product: Streams for Apache Kafka
-:registry-product-long: OpenShift Service Registry
-:registry-product: Service Registry
-:registry: Service Registry
 :cli-product-long: OpenShift Application Services
-// Placeholder URL, when we get a HOST UI for the service we can put it here properly
-:service-url: https://console.redhat.com/application-services/streams/
-:registry-url: https://console.redhat.com/application-services/service-registry/
 :property-file-name: app-services.properties
-
-// Other upstream project names
 :samples-git-repo: https://github.com/redhat-developer/app-services-guides
-
-//URL components for cross refs
-:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/docs/
 :base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
-:getting-started-url: getting-started/README.adoc
-:getting-started-service-registry-url: getting-started-service-registry/README.adoc
-:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
-:kafkacat-url: kcat/README.adoc
-:quarkus-url: quarkus/README.adoc
-:quarkus-service-registry-url: quarkus-service-registry/README.adoc
 :rhoas-cli-url: rhoas-cli/README.adoc
-:rhoas-cli-kafka-url: rhoas-cli-kafka/README.adoc
-:rhoas-cli-service-registry-url: rhoas-cli-service-registry/README.adoc
 :rhoas-cli-ref-url: commands
-:topic-config-url: topic-configuration/README.adoc
-:consumer-config-url: consumer-configuration/README.adoc
 :service-binding-url: service-binding/README.adoc
-:access-mgmt-url: access-mgmt/README.adoc
-:access-mgmt-service-registry-url: access-mgmt-service-registry/README.adoc
-:metrics-monitoring-url: metrics-monitoring/README.adoc
+
+//OpenShift Streams for Apache Kafka
+:kafka-product-long: OpenShift Streams for Apache Kafka
+:kafka-product: Streams for Apache Kafka
+:kafka-product-version: 1
+:service-url: https://console.redhat.com/application-services/streams/
```

Comment:
```suggestion
:kafka-service-url: https://console.redhat.com/application-services/streams/
```

Reply:
For all of the attributes that have "kafka" in the value -- should they also have a "kafka-" prefix in the attribute name?
Particularly if any of the items might potentially have comparable content in other services (for example, access management)?

Reply:
@bredamc Right, I had the same thought but wasn't sure if we should proactively do that for all doc attributes or just the ones that conflict. But good point. Will think on that.